TestSymmetries/par/test_symmetry_xperiodic_yperiodic_zperiodic.par

```diff
@@ -19,9 +19,9 @@ CarpetX::ncells_x = 8
 CarpetX::ncells_y = 8
 CarpetX::ncells_z = 8
 
-CarpetX::periodic_x = yes
-CarpetX::periodic_y = yes
-CarpetX::periodic_z = yes
+Driver::periodic_x = yes
```

Comment:
This seems wrong; `Driver::periodic` is not set.

Reply:
yes, this should be `Driver::periodic = yes`. Hmm, I thought the tests had all passed. Will check.